### ISSUE TITLE
Check for orphaned AUR packages as `yay` does

### ISSUE THREAD

**Issue by Anagastes** [actionless/pikaur#544]
```
Pikaur v1.6.15
Pacman v5.2.2 - libalpm v12.0.2 - pyalpm v0.9.2

```

##### Description:

I switched from yay to pikaur, centrally because of the possibility to clear the cache via HOOK.

However, yay shows packages that no longer exist in a great way.

```bash
:: Searching databases for updates...
:: Searching AUR for updates...
 -> Orphaned AUR Packages:  XXX
 -> Flagged Out Of Date AUR Packages:  XXX
```

Would something similar be possible for pikaur?

**Comment by actionless** [comment]
i'm not interested in such sort of feature, but don't mind it to exist

**Comment by actionless** [comment]
also i don't think just showing a number of those packages gonna be any useful:

i think the optimal way would be to print the list of the packages (like it's currently done for currently installed packages which are not found in AUR), and also to make this feature configurable via pikaur.conf (disabled by default)

**Comment by Anagastes** [comment]
Sorry, I guess I was a bit misleading.

The yay display also shows the individual packages that are not found in the AUR.

> i think the optimal way would be to print the list of the packages (like it's currently done for currently installed packages which are not found in AUR), and also to make this feature configurable via pikaur.conf (disabled by default)

That would be exactly the function, only for AUR packages. I can't find how to activate this for AUR. ^^

**Comment by Anagastes** [comment]
To illustrate what I mean, here is an example. 

```bash
yay -Syu
:: Synchronisiere Paketdatenbanken...
 core ist aktuell
 extra ist aktuell
 community ist aktuell
 multilib ist aktuell
:: Starte vollständige Systemaktualisierung...
 Es gibt nichts zu tun
:: Searching databases for updates...
:: Searching AUR for updates...
 -> Orphaned AUR Packages:  python2-sip-pyqt4
 -> Flagged Out Of Date AUR Packages:  python2-sip-pyqt4
 there is nothing to do
 ```
 With PIKAUR
 
```bash
pikaur -Syu
:: Synchronisiere Paketdatenbanken...
 core ist aktuell
 extra ist aktuell
 community ist aktuell
 multilib ist aktuell

:: Starte komplettes AUR-Upgrade...
Lese Repository-Paketdatenbank...
Lese lokale Paketdatenbank...
Lese AUR Paketinfos...

:: Starte vollständige Systemaktualisierung...
 Es gibt nichts zu tun
```

PS: Sorry, my local language is German

**Comment by actionless** [comment]
>  I can't find how to activate this for AUR. ^^

open text editor and write few python functions :3 or just wait until someone else will do that

**Comment by actionless** [comment]
fixed in #600
